src/components/HomePage.vue

```diff
@@ -2,34 +2,56 @@
   import { ref } from 'vue';
   import ToDoList from './ToDoList.vue';
   import HeaderComponent from './HeaderComponent.vue';
-  import toDoListMT from './../assets/toDoListMT.svg';
+  import toDoListPlaceHolder from './../assets/toDoListPlaceHolder.svg';
+  import PopUpToDoListConfirmation from './popUpToDoListConfirmation.vue';
+
   const toDos = ref([]);
+
   function addNewTodo() {
     const newToDo = {
-      title: 'Title',
-      text: 'tanulj Vue.js-t',
+      title: '',
+      text: 'Cras placerat proin.',
       priority: 'High',
-      created_at: '2023-01-30',
-      status: 'Active',
+      title: 'Title',
+      id: Date.now(),
+      icon: true,
     };
 
     toDos.value.push(newToDo);
   }
   function removeToDo(index) {
+    console.log(index);
     toDos.value.splice(index, 1);
+    console.log(toDos.value);
+  }
+
+  function newDate() {
+    const today = new Date();
+    return (
+      today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate()
```

Comment:
please use template literals for string concatenation, as mentioned in the code style guide.